clients/admin-ui/src/features/privacy-requests/privacy-requests.slice.ts

```diff
@@ -361,7 +361,7 @@ export const privacyRequestApi = createApi({
     }),
     getActiveStorage: build.query<any, void>({
       query: () => ({
-        url: `storage/default/active`,
+        url: `storage/active/default`,
```

Comment:
Are the API model request/response models ready to be generated? Do these API endpoints need invalidateTags such as your PUT API endpoints?